### ISSUE TITLE
Implement AL season 8 changes to character progress

### ISSUE THREAD

**Issue by morepurplemorebetter** [morepurplemorebetter/MPMBs-Character-Record-Sheet#88]
Make the sheet support the changes to how levelling up works in Adventurers League, as it was introduced in AL season 8 (August 2018).

This will require revamping the Adventure Logsheets as well to accommodate for the Treasury Points system as well.

**Comment by tjlanza** [comment]
DDAL is now in Season 9, and has abandoned both treasure points and experience points.

**Comment by morepurplemorebetter** [comment]
> DDAL is now in Season 9, and has abandoned both treasure points and experience points.

You are correct, I was mainly referring to the big change that was introduced with season 8, to no longer use experience points.

Good point about the change in treasure points in season 9, I wasn't aware it had been modified again. I don't play DDAL, so I'm not so good at keeping up with the changes.

Although I still don't fully understand the new season 9 seasonality rules and if that means that when you play content from an old season, rewards follow the rules for that season or the rules for the latest season.

**Comment by tjlanza** [comment]
"Seasonality" as proposed late in Season 8 did not come to pass, but it was about availability of rewards, not rules. DDDAL play always uses the latest rules, even when playing content from prior seasons.

For S9, XP is gone, but when you take a level vs when you got GP is relevant. Treasure Points are no longer a thing, but permanent magic items count is limited by tier.
